src/services/kratos/private.ts

```diff
@@ -22,7 +19,74 @@ export const toDomainSession = (session: KratosSession): Session => {
 
   return {
     id: session.id as SessionId,
-    identity: toDomainIdentityPhone(session.identity),
+    identity: toDomainIdentity(session.identity),
+    expiresAt: new Date(session?.expires_at),
+  }
+}
+
+const toSchema = (schemaId: string): SchemaId => {
+  switch (schemaId) {
+    case "phone_no_password_v0":
+      return SchemaIdType.PhoneNoPasswordV0
+    case "phone_email_no_password_v0":
+      return SchemaIdType.PhoneEmailNoPasswordV0
+    case "email_no_password_v0":
+      return SchemaIdType.EmailNoPasswordV0
+    case "username_password_deviceid_v0":
+      return SchemaIdType.UsernamePasswordDeviceIdV0
+    default:
+      throw new Error(`Invalid schemaId: ${schemaId}`)
```

Comment:
Are you sure we want to throw here? Shouldn't this return a critical `UnknownSchemaError`?

Reply:
it's really to silent type error from typescript: `Function lacks ending return statement and return type does not include 'undefined'.ts(2366)`

it will only arise in production is kratos.yml is misconfigured, so I think that is fair to throw and not return an error. returning an error would have a cascading effect into checking for everything that depend a schema whether there is an error or not, and so this has a lot of boilerplate for really just a type safety that I want to ignore because I know the `switch/case` is meant to be extensive.